populate.js

```diff
@@ -1,5 +1,21 @@
-let books = [];
+let books = class books {
+  constructor() {
+    this.books = [];
+  }
 
+  addBook(book) {
+    this.data.push(book);
+    localStorage.setItem('BOOKS', JSON.stringify(this.book));
+  }
```

Comment:
- [ ] **Required:** Your app doesn't work as it should any longer. Kindly see the gif below for observations

![App is broken](https://user-images.githubusercontent.com/27709832/182576698-06d6cde6-4d57-49e3-bb7f-e067916c2867.gif)
.